src/sentry/static/sentry/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -152,11 +154,14 @@ class SpanTree extends React.Component<PropType> {
 
     const isSpanDisplayed = !isCurrentSpanHidden && !isCurrentSpanFilteredOut;
 
+    const shouldIncludeGap = event.sdk?.name?.includes('javascript') === false;
```

Comment:
@HazAT @kamilogorek is this sufficient?

Reply:
Yes, although official SDK name is `sentry.javascript.{browser,node}`

Reply:
@kamilogorek where is this name established? in relay?

Reply:
https://github.com/getsentry/sentry-javascript/blob/master/packages/browser/src/version.ts
https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/version.ts